MIDAS/lib/ADS7138Q1/ads7138-q1.h

```diff
@@ -1,15 +1,10 @@
 #pragma once
 
-#include <stdint.h>
-// Driver todo
-// https://www.ti.com/product/ADS7138-Q1
+#include<cstdint>
 
-struct ADS7138_1Q {
-    bool init() {
-        return true;
-    }
-    float read_pin(uint16_t pin) {
-        // Dummy code, read from pin
-        return 0.0;
-    }
+struct ADCAddress {
+    int pin_id;
```

Comment:
should this be an struct or a typedef

Reply:
Its a struct so the real analog read and the expander analog read can be differentiated. I chose to have the driver copy the arduino function so its easier to edit other drivers to use the expander, but I'm not sure overloading the function is best overall.